contrib/installer/src/root_command.sh

```diff
@@ -0,0 +1,103 @@
+# Settings
+source_url=${args[source]}
+publisher=${args[publisher]}
+# Flags
+no_call=${args[--no-call]}
+dry_run=${args[--dry-run]}
+
+REQS="wget apt-transport-https"
+
+function do_call() {
+    if [[ $dry_run ]]; then
+        echo -e "### DRY RUN: \n$1"
+    else
+        $1
+    fi
+}
+
+function get_distribution {
```

Comment:
@matmair I see this is a duplicate of `contrib/installer/install` - not sure if there is a way around this thought? LMK what you think

Reply:
@SchrodingersGat this is intended. `contrib/installer/install` is generated by combining the root command and all the boilerplate to get a nice CLI interface, selectable options etc. Then there is a symlink between `contrib/installer/install` and `contrib/install.sh` to keep the path as short as possible (and backwards compatible).

There is a link to the used [tool](https://bashly.dannyb.co/installation/) in the README `contrib/installer/README`

Reply:
@matmair fair enough, I'm happy with this then